src/lib/emscripten.ts

```diff
@@ -4,7 +4,7 @@ type EmscriptenSymbolMap = Map<string, string>
 //
 // NOTE: This will fail to ignore escaped backslahes (e.g. `\\20`).
 function unescapeHex(input: string): string {
-  return input.replace(/\\([a-fA-F0-9]{2})/, (_match, group) => {
+  return input.replace(/\\([a-fA-F0-9]{2})/g, (_match, group) => {
```

Comment:
Eugh, JavaScript regexes are the worst regexes. Why is the regex stateful :<

Reply:
😢